src/main/java/kafdrop/controller/MessageController.java

```diff
@@ -249,6 +255,10 @@ public String addMessage(
     String topicName,
     @ModelAttribute("addMessageForm") CreateMessageVO body,
     Model model) {
+    if (!sendEnabled) {
+      model.addAttribute("errorMessage", "Message sending is not enabled.");
+      return "message-inspector";
+    }
     try {
```

Comment:
To prevent this early return with the repeated return value, I suggest this approach:
```suggestion
    try {
      if (!sendEnabled) {
        throw new IllegalStateException("Message sending is not enabled.");
      }
```

Reply:
I'm addressed it